force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,17 +77,25 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+        List<String> words = sentence.split(' ', 0);
+        Set<String> listOfWords = new Set<String>();
+        
+        for (String s1 : words) {
+            listOfWords.add(s1);            
+        } String joinedList = String.join(new List<String>(listOfWords), ' ');
+         
         // Create a set to store the unique words
 
         // Add each word to the set
 
         // Join the unique words back into a sentence
 
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return joinedList; // Replace null with the variable you used to store the result
```

Comment:
Two things here. 

1. You can add list to set directly and skip iteration
2. Do name a Set datatype ...List. That is confusing. They are very different. 

```
 public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
```